tests/gem5/verifier.py

```diff
@@ -106,6 +106,29 @@ def _generic_instance_warning(self, kwargs):
                              ' generic %s'
                              ' instead' % MatchGoldStandard.__name__)
 
+class MatchStat(MatchGoldStandard):
```

Comment:
Let's call this match stats since you can match more than 1

Also, it would be good to add a brief docstring. Maybe something like:

```python
"""
For a list of stats, this verifier checks if any of them have changed. If so, the test fails.
This class does a simple diff like MatchGoldStandard, but then only checks for the stats
in the diff. This allows some stats to change (e.g., host stats) without causing the test to fail.
"""
```